blog/2015-05-27-so-you-want-to-transform-part-1.adoc

```diff
@@ -0,0 +1,215 @@
+= So You Want to Transform Data? Fuse Transformation tooling, part 1
+
+:page-layout: blog
+:page-author: bfitzpat
+:page-tags: [fuse, transformation, jbosstools]
+
+We've been working hard on improving our tooling story for Fuse for a
+few months. One of the areas we've been focusing on specifically has
+been Transformation tooling. It's been missing in the JBoss Tools suite
+for a while now and something we needed to address.
+
+As a result, we ended up with the Fuse Transformation tooling, which
+I'll be talking about over the course of a few blog posts.
+
+== First, What Exactly _Is_ the Fuse Transformation Tooling?
+
+The last transformation tooling we had in JBoss Tools leveraged Smooks
+to convert one format to another. This time we're using the Dozer
+framework, which offers a great base for handling Java to Java
```

Comment:
this intro is covering a lot of techs - maybe add links to some o fthem ? like Dozer and Fuse ?